### DISCUSSION TITLE
Changing titlestring in options.lua doesn't work

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [LazyVim/LazyVim#1850] [Q&A] [answered]
Title.

options.lua:
```
local opt = vim.opt
opt.titlestring = "%t - NVIM"

```

**Comment by limaceous-bushwhacker** [comment, marked as answer]
Need to enable title first (`opt.title = true`)

**Comment by folke** [comment]
fyi: I didn't enable title by default, since it used to give weird issues on some Wayland compositors like Gnome
